lib/sysauthtest.pm

```diff
@@ -0,0 +1,5 @@
+package sysauthtest;
+use base "opensusebasetest";
```

Comment:
we actually stopped using empty baseclasses. If there is nothing to share, just use opensusebasetest as your base